src/project-converter.ts

```diff
@@ -203,4 +203,16 @@ export class ProjectConverter {
 
     return results;
   }
+
+  /**
+   * Updates polymer cli and wct usages to work with js modules
+   */
+  convertTravisYaml(travisYaml: string) {
+    return travisYaml
+        .replace(/polymer\ install.*/g, 'yarn')  // polymer install => yarn
```

Comment:
these replacements seem a little aggressive. Is there a way to limit them to the fields we expect to see them in, or at least to values, not keys?

Reply:
@justinfagnani Yeah, I was thinking the same thing. When I started looking at more `.travis.yml` files, I had some open questions that maybe you can help out with?